### PR TITLE
MOTECH-2907: hotfix visit is now optional in encounter action

### PR DESCRIPTION
**Pull request by palbecki**
- visit is now optional in EventHelper.encounterParameters

### CODE REVIEW

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

File: openmrs/src/main/java/org/motechproject/openmrs/tasks/constants/DisplayNames.java [hunk 1]
```diff
@@ -50,7 +50,7 @@
     public static final String VISIT_UUID = "openMRS.visit.uuid";
```
Comment (kmadej) [inline openmrs/src/main/java/org/motechproject/openmrs/tasks/constants/DisplayNames.java:50]: this changes to properties are not related with this ticket, please leave in this PR only changes in EventHelper class

**Review by kmadej — CHANGES_REQUESTED** [review]

**Review by kmadej — APPROVED** [review]
